src/components/form-elements/Textarea.jsx

```diff
@@ -123,13 +130,34 @@ const Textarea = (props: TextareaPropsType) => {
     className
   );
 
+  const errorMessageClass = classnames(
+    'sg-textarea__error-message',
+    {
+      [`sg-textarea__error-message--${String(size)}`]: size !== SIZE.NORMAL,
+    },
+    className
+  );
+
   return (
-    <Type
-      {...additionalProps}
-      className={textareaClass}
-      ref={textareaRef}
-      value={value}
-    />
+    <>
+      <Type
+        {...additionalProps}
+        className={textareaClass}
+        ref={textareaRef}
+        value={value}
+      />
+      {errorMessage !== undefined &&
+        errorMessage !== null &&
+        errorMessage !== '' && (
+          <Text
+            size={size === SIZE.NORMAL ? 'xsmall' : 'small'}
+            color="peach-dark"
+            className={errorMessageClass}
```

Comment:
maybe we could achieve all of this using `<Flex />` here?

Reply:
Yes! I'll change it ASAP

Reply:
done